builtInFunctions/esdtNFTTransfer.go

```diff
@@ -296,7 +296,12 @@ func (e *esdtNFTTransfer) processNFTTransferOnSenderShard(
 			return nil, err
 		}
 	} else {
-		err = e.esdtStorageHandler.AddToLiquiditySystemAcc(esdtTokenKey, esdtData.Type, nonce, big.NewInt(0).Neg(quantityToTransfer))
+		hasDynamicRole, err := hasDynamicRole(acntSnd, tickerID, e.marshaller)
```

Comment:
to improve the understanding of the code I would have named the `hasDynamicRole` to `keepMetadataOnZeroLiquidity`
Reason: we all know how the variable is populated (by calling the function hasDynamicRole) but is more obscure to understand what is the parameter of the AddToLiquiditySystemAcc function

Reply:
Good idea, renamed.